src/App.js

```diff
@@ -24,11 +24,10 @@ function App() {
 			<Route path="/" element={<ProtectedRoute />}>
 				<Route index element={<Dashboard />} />
 				<Route path="dashboard" element={<Dashboard />} />
-				<Route path="questTwo" element={<Quest2 />} />
 				<Route path="profile" element={<Profile />} />
 				<Route path="logout" element={<Logout />} />
 				<Route path="quests-menu" element={<QuestsMenu />} />
-				<Route path="quest2" element={<Quest2 />} />
+				<Route path="quest1" element={<Quest2 />} />
```

Comment:
This change is made for testing the quest functionality from quests menu, after we add quest1, this should be changed